content/200-concepts/300-more/050-environment-variables/index.mdx

```diff
@@ -7,18 +7,29 @@ tocDepth: 2
 
 <TopBlock>
 
-You can use environment variables to provide configuration options when a CLI command is invoked. Environment variables allow you to:
+An environment variable is a key value pair stored on your machines local environment. Traditionally the name of the variable is uppercase, this is then followed by an equals sign then the value of the variable.
 
-- Keep secrets out of the schema file
-- Improve portability of the schema file
-
-You can use machine-specific variables, `.env` files, or a combination of both.
+```env
+MY_VALUE=prisma
+```
 
```

Comment:
Here we could add a bit of wikipedia like information what environment variables are usually used for:

> An environment variable is a dynamic-named value that can affect the way running processes will behave on a computer. They are part of the environment in which a process runs. For example, a running process can query the value of the TEMP environment variable to discover a suitable location to store temporary files, or the HOME or USERPROFILE variable to find the directory structure owned by the user running the process.

The `TEMP` or `HOME` is a great example as that will make intuitive sense for developers.